docs/id/docs/index.md

```diff
@@ -224,44 +224,44 @@ The command `uvicorn main:app` refers to:
 
 </details>
 
-### Check it
+### Periksa
 
-Open your browser at <a href="http://127.0.0.1:8000/items/5?q=somequery" class="external-link" target="_blank">http://127.0.0.1:8000/items/5?q=somequery</a>.
+Buka browser anda pada <a href="http://127.0.0.1:8000/items/5?q=somequery" class="external-link" target="_blank">http://127.0.0.1:8000/items/5?q=somequery</a>.
 
-You will see the JSON response as:
+Kamu akan melihat response JSON sebagai berikut:
 
 ```JSON
 {"item_id": 5, "q": "somequery"}
 ```
 
-You already created an API that:
+Kamu telah berhasil membuat API yang memiliki kemampuan dibawah ini:
 
-* Receives HTTP requests in the _paths_ `/` and `/items/{item_id}`.
-* Both _paths_ take `GET` <em>operations</em> (also known as HTTP _methods_).
-* The _path_ `/items/{item_id}` has a _path parameter_ `item_id` that should be an `int`.
-* The _path_ `/items/{item_id}` has an optional `str` _query parameter_ `q`.
+* Menerima HTTP requests pada _jalur_ `/` dan _jalur_  `/items/{item_id}`.
```

Comment:
why not keep it 'path' instead of translate it? just like how you dont translate 'query'. I think people will understand

Reply:
Yes, actually I wanted to keep it as 'path' but I changed it to 'jalur' after I read the google glossary that mentions 'jalur' as a path in the Indonesian language. 
https://support.google.com/google-ads/answer/14095?hl=id

Reply:
Okay, after I have discussed it with friends, I will do like you suggest to keep 'path'. So, I think it's okay to mark this as resolved. Thank you @negadive.